### DISCUSSION TITLE
update extension with esbuild then fail to set breakpoint in source file while running extension? what is wrong?

### DISCUSSION THREAD

**Discussion by xkeshav** [microsoft/vscode-discussions#2044] [Extension Development QnA] [answered]
built a vs code extension and it is on marketplace . now want to update it with esbuild; build is working finre but now I am unable to put breakpoint in source file while running the extesnion host to test the extension locally.

earlier ( without esbuild ) I have this things easy ; just to click on debug and click on Run extension 

there is no change in launch.json but unable to make a breakpoint; 
someone can help me how to debug with esbuild; source map also generated.

you can check the code

without esbuild: [main branch](https://github.com/xkeshav/color-collector/tree/main)
with esbuild: [feature/esbuild branch](https://github.com/xkeshav/color-collector/blob/feature/esbuild/.vscode/launch.json)

**Comment by isidorn** [comment]
Maybe @bpasero has some suggestions

**Reply by bpasero** [reply]
I think @connor4312 would know, I doubt its related to our ESM work.

**Comment by connor4312** [comment, marked as answer]
In your lauch config, your outFiles are pointing to `dist` but your build actually generates output in `out`.

You can just remove your `outFiles` entry and the debugger should figure out the right ones automatically.
